doc/releases/changelog-dev.md

```diff
@@ -21,6 +21,10 @@
 
 * `SProd.terms` now flattens out the terms if the base is a multi-term observable.
   [(#5885)](https://github.com/PennyLaneAI/pennylane/pull/5885)
+  
+* New functionality has been added to natively support exponential extrapolation when using the `mitigate_with_zne`. This allows
+  users to have more control over the error mitigation protocol without needing to add further dependencies.
+  [(#5972)](https://github.com/PennyLaneAI/pennylane/pull/5972)
 
```

Comment:
@natestemen Please add yourself in the list of contributors :)

Reply:
https://github.com/PennyLaneAI/pennylane/pull/5972/commits/e8c95e7cdbc71ae4bee4d1078b40410940d9ac49